mlflow/server/js/package.json

```
@@ -138,8 +136,8 @@
     "extract-i18n:base": "node scripts/extract-i18n.js",
     "extract-i18n:check": "yarn extract-i18n:base --lint",
     "extract-i18n": "yarn extract-i18n:base",
-    "lint": "eslint src",
-    "lint:fix": "eslint src --fix",
+    "lint": "eslint --ext js,jsx,ts,tsx src",
+    "lint:fix": "eslint--ext js,jsx,ts,tsx src --fix",
```

```suggestion
    "lint:fix": "eslint --ext js,jsx,ts,tsx src --fix",
```

This line needs to be fixed. I'll open a PR.